backgroundJobs/Addresses.js

```diff
@@ -126,10 +133,67 @@ const tectonicAddress = { "CRO" : { "comptroller" : "0xb3831584acb95ED9cCb0C11f6
     "deployBlock" : 570286, "blockStepInInit" : 2000, "multicallSize" : 20 }
 }
 
+///////////// AAVE V3 SPECIFICS //////////////////////////
+
+const aaveV3Configuration =
+{
+	"AVAX": {
+		"poolAddressesProviderRegistry": "0x770ef9f4fe897e59daCc474EF11238303F9552b6",
+		"deployBlock": 11970175,
+		"blockStepInInit": 1000,
+        "firstEventBlock": 12027175,
```

Comment:
fix indentation, also below

Reply:
Indentation is good locally, github probably display it wrong